presentation/src/main/java/me/androidbox/presentation/task/screen/TaskDetailScreen.kt

```diff
@@ -0,0 +1,184 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.maxkeppeker.sheets.core.models.base.rememberUseCaseState
+import me.androidbox.component.agenda.AgendaDetailTopBar
+import me.androidbox.component.agenda.AgendaDurationSimple
+import me.androidbox.component.agenda.AgendaHeader
+import me.androidbox.component.agenda.AgendaHeaderItem
+import me.androidbox.component.agenda.AlarmReminder
+import me.androidbox.component.agenda.AlarmReminderItem
+import me.androidbox.component.agenda.EditModeType
+import me.androidbox.component.general.AgendaDropDownMenu
+import me.androidbox.component.ui.theme.Black
+import me.androidbox.component.ui.theme.White
+import me.androidbox.component.ui.theme.backgroundBackColor
+import me.androidbox.component.ui.theme.backgroundWhiteColor
+import me.androidbox.component.ui.theme.dropDownMenuBackgroundColor
+import me.androidbox.domain.DateTimeFormatterProvider.DATE_PATTERN
+import me.androidbox.domain.DateTimeFormatterProvider.LONG_DATE_PATTERN
+import me.androidbox.domain.DateTimeFormatterProvider.TIME_PATTERN
+import me.androidbox.domain.DateTimeFormatterProvider.formatDateTime
+import me.androidbox.presentation.ui.theme.BusbyTaskyTheme
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TaskDetailScreen(
+    taskDetailScreenState: TaskDetailScreenState,
+    taskDetailScreenEvent: (taskDetailScreenEvent: TaskDetailScreenEvent) -> Unit,
+    onEditTitleClicked: (title: String) -> Unit,
+    onEditDescriptionClicked: (description: String) -> Unit,
+    onCloseClicked: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val calendarStateTimeDate = rememberUseCaseState()
+
+    Scaffold(
+        modifier = modifier,
+        topBar = {
+            AgendaDetailTopBar(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .background(color = MaterialTheme.colorScheme.backgroundBackColor)
+                    .padding(horizontal = 16.dp),
+                editModeType = EditModeType.SaveMode(),
+                displayDate = taskDetailScreenState.from.formatDateTime(LONG_DATE_PATTERN),
+                onCloseClicked = {
+                    onCloseClicked()
+                },
+                onEditClicked = {
+
+                },
+                onSaveClicked = {
+
+                })
+        }) { paddingValues ->
+
+        Column(modifier = Modifier
```

Comment:
Whats the use of this Column inside another column?

Reply:
@MKiperszmid I was using it to create the background 
![task-screen](https://user-images.githubusercontent.com/14260802/235589700-d62b5547-7df8-477d-874b-1174ca87a57a.png)

Reply:
@MKiperszmid I was using it to create the background